readme/dev/accessibility.md

```diff
@@ -0,0 +1,91 @@
+# Development: Accessibility
+
+Joplin has a strong focus on accessibility. It's important to make sure that new pull requests and features keep Joplin accessible.
+
+## Making new components accessible
+
+When creating new components, it's important to make sure that they're accessible. How to do this varies a bit between desktop and mobile, but in general:
+- **Labels**: All focusable controls should have labels. Usually, these labels are provided by text inside the controls. Components that don't have any text may need to be labelled manually.
+- **Minimum size**: Buttons should be at least 24px by 24px.
+- **Keyboard accessibility**: It shouldn't be necessary to use a mouse to use a component.
+- **Screen reader accessibility**: It should be possible to use the desktop and mobile apps with a [screen reader](https://en.wikipedia.org/wiki/Screen_reader).
+- **Contrast**: Text and images should have high contrast with the background (see [the WebAIM contrast checker](https://webaim.org/resources/contrastchecker/)).
```

Comment:
I would suggest making these H3 headers and give more details. Maybe some general pointers (for example "if a focusable control doesn't have a label, set an aria-label instead" or links that explains more precisely what to do for that particular section. We can't cover everything here but there are certain things that need to be done more often than not and those things we can mention

Reply:
I've replaced most of this information with links to the relevant WCAG 2.2 "Understanding" docs.